File: test/9b45633a1769c5/swift/collatz.swift
func collatz(_ j: Int) -> Int {
    var cnt: Int = 0
    var i = j
    while (i != 1) {
        cnt += 1
        if (i % 2 == 0) {
            i /= 2
        } else {
            i *= 3
            i += 1
        }
    }
    return cnt
}
for i in 0...1000000000000000000 {
    print(collatz(i))
}

File: test/9b45633a1769c5/go/collatz.go
package main

import "fmt"

func main() {
	for i := 0; i < 1000000000000000000; i++ {
		fmt.Println(collatz(i))
	}
}
func collatz(j int) int {
	i := j
	cnt := 0
	for i != 1 {
		cnt++
		if i%2 == 0 {
			i /= 2
		} else {
			i *= 3
			i += 1
		}
	}
	return cnt
}
